legal-api/src/legal_api/resources/v2/business/business_documents.py

```diff
@@ -76,41 +77,56 @@ def _get_document_list(business):
-
-    if coop_rules_document:
-        coop_rules_filing = Filing.find_by_id(coop_rules_document.filing_id)
-        coop_rules_doc_url = url_for('API2.get_documents', **{'identifier': business_identifier,
-                                                              'filing_id': coop_rules_filing.id,
+    rules_document = Document.find_by_business_id_and_type(business_id, DocumentType.COOP_RULES.value)
+
+    if rules_document:
+        rules_filing = Filing.find_by_id(rules_document.filing_id)
+        rules_doc_url = url_for('API2.get_documents', **{'identifier': business_identifier,
+                                                         'filing_id': rules_filing.id,
+                                                         'legal_filing_name': None})
+        documents['certifiedRules'] = f'{base_url}{rules_doc_url}/certifiedRules'
+        info['certifiedRules'] = {
+            'key': rules_document.file_key,
+            'name': DocumentMetaService.general_filename(
+                rules_filing.filing_date.isoformat(), business.identifier, 'Certified Rules'),
+            'uploaded': rules_document.uploaded.isoformat() if rules_document.uploaded else None
+        }
+
+    memorandum_document = Document.find_by_business_id_and_type(business_id, DocumentType.COOP_MEMORANDUM.value)
+
+    if memorandum_document:
+        memorandum_filing = Filing.find_by_id(memorandum_document.filing_id)
+        memorandum_doc_url = url_for('API2.get_documents', **{'identifier': business_identifier,
+                                                              'filing_id': memorandum_filing.id,
                                                               'legal_filing_name': None})
-        coop_documents['certifiedRules'] = f'{base_url}{coop_rules_doc_url}/certifiedRules'
-
-    coop_memorandum_document = Document.find_by_business_id_and_type(business_id, DocumentType.COOP_MEMORANDUM.value)
-
-    if coop_memorandum_document:
-        coop_memorandum_filing = Filing.find_by_id(coop_memorandum_document.filing_id)
-        coop_memorandum_doc_url = url_for('API2.get_documents', **{'identifier': business_identifier,
-                                                                   'filing_id': coop_memorandum_filing.id,
-                                                                   'legal_filing_name': None})
-        coop_documents['certifiedMemorandum'] = f'{base_url}{coop_memorandum_doc_url}/certifiedMemorandum'
-
-    return coop_documents
+        documents['certifiedMemorandum'] = f'{base_url}{memorandum_doc_url}/certifiedMemorandum'
+        info['certifiedMemorandum'] = {
+            'key':  memorandum_document.file_key,
+            'name': DocumentMetaService.general_filename(
+                memorandum_filing.filing_date.isoformat(), business.identifier, 'Certified Memorandum'),
+            'uploaded': memorandum_document.uploaded.isoformat() if memorandum_document.uploaded else None
```

Comment:
@argush3 let me know if this is ok. Technically we could just use `'uploaded': memorandum_filing.filing_date.isoformat()`

Reply:
yeah maybe just using `memorandum_filing.filing_date.isoformat()` is good enough for now.  Then we don't need `created` and `uploaded` columns in `documents` table.  If the business ever really wants the true "uploaded" date, we can add `uploaded` column to `documents` table then and backfill the uploaded times using minio.

Reply:
Sounds good I'll make the change